src/constants/OfferType.ts

```diff
@@ -5,7 +5,7 @@ const OfferTypes = {
   PLACES: 'places',
 } as const;
 
-const ScopeTypes = { ...OfferTypes, ORGANIZERS: 'organizers' };
+const ScopeTypes = { ...OfferTypes, ORGANIZERS: 'organizers' } as const;
```

Comment:
These seems like changes from main

If you would merge main into `feature/III-6557` these changes would probably disappear?